openmetadata-ui/src/main/resources/ui/src/utils/TableTags/TableTags.utils.tsx

```diff
@@ -0,0 +1,29 @@
+/*
+ *  Copyright 2022 Collate.
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import { TableTagsProps } from 'components/TableTags/TableTags.interface';
+import { TagLabel, TagSource } from 'generated/type/tagLabel';
+import { reduce } from 'lodash';
+
+export const getFilterTags = (tags: TagLabel[]): TableTagsProps =>
+  reduce(
+    tags,
+    (acc, cv) => {
+      if (cv.source === TagSource.Classification) {
+        return { ...acc, Classification: [...acc.Classification, cv] };
+      } else {
+        return { ...acc, Glossary: [...acc.Glossary, cv] };
+      }
+    },
+    { Classification: [] as TagLabel[], Glossary: [] as TagLabel[] }
+  );
```

Comment:
We can simply use groupBy from lodash here
```
groupBy(tags, 'source') // {"Classification": TagLabel[], "Glossary": TagLabel[]
```

Reply:
@chirag-madlani We need this as not all the time glossary or  tag will be there, so no type key will be created by their source name and we required it for further process.